src/uu/tail/src/tail.rs

```diff
@@ -54,73 +58,80 @@ impl Default for Settings {
 
 #[allow(clippy::cognitive_complexity)]
 pub fn uumain(args: impl uucore::Args) -> i32 {
-    let args = args.collect_str();
+    let program = args.next().unwrap_or_else(|| OsString::from(NAME));
+    let args = std::iter::once(program.clone()).chain(args);
```

Comment:
I don't think it is needed (both program & args)

Reply:
Removed both.